skill/api.py

```diff
@@ -77,15 +77,37 @@
 )
 
 
+def own_language(lang: str):
```

Comment:
So tiny - but is this meant to be `owm_language` as in the language for open weather maps